.github/workflows/docker.yaml

```diff
@@ -0,0 +1,36 @@
+name: Create and publish a Docker image
```

Comment:
The currently workflow I follow is to publish a release and Github will host that version for us. What's the intent behind having the docker image workflow? Is this more relevant for gitlab?